test.md

```diff
@@ -0,0 +1 @@
+# This file should be ignored by git
```

Comment:
- [ ] Also consider adding `.gitignore` file and add this file inside it beside `node_modules`

Reply:
- [ ] Kindly add `.gitignore` and not` random.gitignore` just as the previous reviewer suggested 👍🏼 .